samples/charts/pie-chart/legend/Pages/PieChartLegend.razor

```diff
@@ -2,28 +2,31 @@
 @using IgniteUI.Blazor.Controls
 @inject IIgniteUIBlazor IgniteUIBlazor
 
-<div class="igContainer">
+    <div class="igContainer">
+        <span class="igLegend-title">Global Electricity Demand by Energy Use</span>
+        <div class="igLegend">
+            <ItemLegend @ref="Legend" />
```

Comment:
check if Legend instead of ItemLegend works with PieChart
 <Legend @ref="Legend" Orientation="LegendOrientation.Horizontal"/>